contracts/cw1-subkeys/src/contract.rs

```diff
@@ -274,6 +275,26 @@ impl Cw1SubkeysContract<'_> {
         })
     }
 
+    #[msg(migrate)]
+    fn migrate_0_1_to_0_2(
```

Comment:
I will name this just migrate. In migrate, one can then dispatch between migrations for specific versions (to we may later do it automagically I think)